### PR TITLE
Use recurly docs

### PR DESCRIPTION
**Pull request by dbrudner**

### CODE REVIEW

File: docs/2-hooks/useRecurly.stories.mdx [hunk 1]
```diff
@@ -39,6 +41,56 @@ export function MyCardForm (props) {
...
+      console.log('Cancelled');
+    });
+
+    paypal.on('ready', () => {
+      console.log('Ready');
+    });
+
+    // Remove listeners on unmount
+    // See https://reactjs.org/docs/hooks-effect.html#effects-with-cleanup for more info.
+    return () => {
```
Comment (dbrudner) [inline docs/2-hooks/useRecurly.stories.mdx]: This might be overkill for an example.

File: docs/2-hooks/useCheckoutPricing.stories.mdx [hunk 2]
```diff
@@ -9,7 +9,9 @@ import { PropsTable } from '@storybook/components';
 
 `import { useCheckoutPricing } from '@recurly/react-recurly';`
 
-A [custom hook][react-hooks] for interacting with [Recurly.js' pricing API][rjs-pricing] for providing users with an estimate of a purchase before they check out.
+Use this [hook][react-hooks] to interact with [Recurly.js' pricing API][rjs-pricing] and provide users with an estimate of a purchase before they check out.
```
Comment (chrissrogers) [inline docs/2-hooks/useCheckoutPricing.stories.mdx]: how about "the Recurly.js pricing API"?

File: docs/2-hooks/useRecurly.stories.mdx [hunk 3]
```diff
@@ -3,11 +3,13 @@ import { Subtitle } from '@storybook/addon-docs/blocks';
...
 # useRecurly
-<Subtitle slot={() => 'To get a token'} />
+<Subtitle slot={() => 'To use a Recurly instance'} />
 
 `import { useRecurly } from '@recurly/react-recurly';`
 
-Use this [hook][react-hooks] to generate a token.
+Use this [hook][react-hooks] to access a Recurly instance.
+
+## Token
```
Comment (chrissrogers) [inline docs/2-hooks/useRecurly.stories.mdx]: How about "Getting a token" ?

File: docs/2-hooks/useRecurly.stories.mdx (hunk 1 above)
Comment (chrissrogers) [reply docs/2-hooks/useRecurly.stories.mdx]: Yeah I think it's okay to leave this off. We should distill it down to a `PayPal#destroy` method to perform the cleanup then document that usage here once that's available.

File: docs/2-hooks/useRecurly.stories.mdx [hunk 4]
```diff
@@ -39,6 +41,54 @@ export function MyCardForm (props) {
 }
 ```
 
+## PayPal
```
Comment (chrissrogers) [inline docs/2-hooks/useRecurly.stories.mdx]: How about "Extended usage example: PayPal" ?

File: docs/2-hooks/useRecurly.stories.mdx [hunk 5]
```diff
@@ -39,6 +41,54 @@ export function MyCardForm (props) {
...
+## Extended usage example: PayPal
+
+`useRecurly` can be used to expose any functionality in [Recurly-js][recurly-js].
+
+The example below shows how to use `useRecurly` to interact with the [Recurly-js PayPal API][paypal].
+
+```jsx
+const PayPalButton = () => {
+  const { recurly } = useRecurly();
+  const paypal = recurly.PayPal();
```
Comment (chrissrogers) [inline docs/2-hooks/useRecurly.stories.mdx]: nit: `payPal` 🚀

**Pull request merged**